src/applications/vaos/tests/mocks/helpers.js

```diff
@@ -45,7 +46,7 @@ export function mockAppointmentInfo({
   setFetchJSONResponse(
     global.fetch.withArgs(
       `${environment.API_URL}/vaos/v0/appointment_requests?start_date=${moment()
-        .add(-30, 'days')
+        .add(isHomepageRefresh ? -120 : -30, 'days')
```

Comment:
The new appointment requests list goes back 120 days instead of 30.  In order to prevent old tests mocks from breaking, I've added the `isHomepageRefresh` flag which will toggle between -120 and -30 days.